server/lib/job-queue/handlers/video-upload-tmp-folder-cleaner.ts

```diff
@@ -0,0 +1,53 @@
+import { readdir, unlink, stat } from 'fs-extra'
+import { Stats } from 'node:fs'
+import * as path from 'path'
+import { logger } from '@server/helpers/logger'
+import { getTmpPath } from '@server/helpers/utils'
+import { SCHEDULER_INTERVALS_MS } from '@server/initializers/constants'
+
+async function processVideoUploadTmpFolderCleaner () {
+  const tmpPath = await getTmpPath()
+  let files: string[]
+
+  logger.debug('processVideoUploadTmpFolderCleaner: Reading files from path %s', tmpPath)
+
+  try {
+    files = await readdir(tmpPath)
+  } catch (error) {
+    logger.error('Failed to read resumable video upload tmp folder.', { error })
+    return
+  }
+
+  logger.debug('processVideoUploadTmpFolderCleaner: Found %d files', files.length)
+
+  await Promise.all(files.map(async function (file) {
+    const filePath = path.join(tmpPath, file)
+    let statResult: Stats
+
+    try {
```

Comment:
Wrap the entire map function in a try/catch since you don't recover from the error